### PR TITLE
Fix addRow and addRows method

### PR DESCRIPTION
**Pull request by sapics**
Sorry, I have not tested this PR yet.
I would like to use addRow method with v3, thus, I make this PR with #302 which is created by @crisdanilo.

@crisdanilo Could you check this fix is what you proposed?
@theoephraim Thank you very much for updating to v3!!

### CODE REVIEW

**Comment by crisdanilo** [comment]
Thank you @sapics for the fix!!

File: lib/GoogleSpreadsheetWorksheet.js [hunk 1]
```diff
@@ -358,7 +358,7 @@ class GoogleSpreadsheetWorksheet {
       url: `/values/${encodeURIComponent(this.a1SheetName)}:append`,
       params: {
         valueInputOption: 'USER_ENTERED', // RAW
```
Comment (crisdanilo) [inline lib/GoogleSpreadsheetWorksheet.js]: Here, it's necessary change the value to `RAW`,  otherwise it could not work

**Comment by sapics** [comment]
@crisdanilo Thank you very much for reviewing and teaching the correction!
I fixed my commit.

**Comment by theoephraim** [comment]
I did not change the default behaviour as I think it handles most situations in the expected manner. I think overwriting empty rows makes sense and it handles adding rows if required. I also think when you're messing with rows, you probably want to use the "user entered" interface, which is the same as if you typed it into the sheets ui.

But, I added these things as options in https://github.com/theoephraim/node-google-spreadsheet/commit/a6baa65a327d0265e8af090b994b9439c7ac44ff

you can now pass an options object to addRow and addRows
for example: `sheet.addRow(rowdata, { raw: true, insert: true })`

you can also pass this options object when saving a row to use the raw values.
example: `row.save({ raw: true })`

**Comment by sapics** [comment]
Thanks for adding options. It looks good!

**Pull request closed without merging**